cmd/db/dynamodb.go

```diff
@@ -39,16 +40,20 @@ func DB() *dynamodb.DynamoDB {
 
 func main() {
 	var i int
-	fmt.Print("(1) Create DP Table:\n(2) Delete DP Table: ")
-	_, err := fmt.Scanf("%d", &i)
-	if err != nil {
-		fmt.Println(err)
-	}
-	switch i {
-	case 1:
-		CreateTable()
-	case 2:
-		DeleteTable()
+	for {
+		fmt.Print("(1) Create DP Table:\n(2) Delete DP Table:\n(0) Quit\n")
+		_, err := fmt.Scanf("%d", &i)
+		if err != nil {
+			fmt.Println(err)
+		}
+		switch i {
+		case 1:
+			CreateTable()
+		case 2:
+			DeleteTable()
+		default:
```

Comment:
this does essentially the same as not having the loop; I meant to have a loop such that allowed you to "try again" if you made a mistake in the input (say, pressed '12' or '`' by mistake.)

Reply:
Oh is that what you meant? I don't really see the need for that kind of error checking?

Reply:
This loop allows you to create and delete databases in one running session.

Reply:
In any case ^ I have updated the code to reflect the result you desired